src/Badge/Badge.svelte

```diff
@@ -26,13 +38,40 @@
    */
   export let href = '';
 
+  /**
+   * Create a circular indicator for absolute positioned badge.
+   * @type {string}
```

Comment:
Shouldn't this be `@type {boolean}` too?

Reply:
Yes, it should. Copy pasta 🍝